src/sardana/tango/core/SardanaDevice.py

```diff
@@ -512,7 +512,7 @@ def _get_class_properties(self):
         return dict(ProjectTitle="Sardana", Description="Generic description",
                     doc_url="http://sardana-controls.org/",
                     __icon=self.get_name().lower() + ".png",
-                    InheritedFrom=["Device_4Impl"])
+                    InheritedFrom=["Device_5Impl"])
```

Comment:
Maybe this to not hard-code it could be `LatestDeviceImpl.__name__`? or sth like this? Anyway if its difficult to do don't worry. I think that in the HLAPI we could avoid using `DeviceClass`, right?

Reply:
It would be easy if `LatestDeviceImpl` was an alias for `Device_5Impl`. The PyTango implementation is actually a subclass of `Device_5Impl` (I cannot remember why) so to find a name we are looking for, we would have to fetch the super class and get its name.

Taking into account that:
* this information is not actually used by anyone 
* we will probably have a HLAPI sardana implementation much sooner than a Device_6Impl. (and anyway a PyTango 10 will probably not expose the low level API anyway ;-)

I don't think it is worth the effort

Reply:
I agree, I commented just in case it was easy to do.